src/bugsnag/client/api/base.ts

```diff
@@ -72,16 +89,18 @@ export class BaseAPI {
         headers: response.headers
       }
 
+      // Just to make sure the server handles rate limiting properly
+      if (response.status === 429) {
```

Comment:
Can we remove this? It's not a good idea to just sit and wait inside the request if we get rate limited. To the user it will just seem like the MCP server is frozen/unresponsive.